.changeset/forty-carpets-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration-aws-node': patch
```

Comment:
If we update this to minor and change the version in package.json to 0.0.0 the first released version will be `0.1.0`

```suggestion
'@backstage/integration-aws-node': minor
```